docs/events/yield_events.md

```diff
@@ -61,3 +60,45 @@ def progress_example():
     )
 
 ```
+
+## Yielding Other Events
+
+Events can also yield other events. This is useful when you want to chain events together. To do this, you can yield the event handler function itself.
+```md alert
+# Reference other Event Handler via class
+
+When chaining another event handler with `yield`, access it via the state class, not `self`.
+```python demo exec
+
+import asyncio
+
+class YieldEventsState(rx.State):
+    count: int = 0
+    show_progress: bool = False
+
+    async def add_five(self):
+        self.show_progress = True
+        yield
+        # Think really hard.
+        await asyncio.sleep(1)
+        self.count += 5
+        self.show_progress = False
+
+    async def increment(self):
+        yield YieldEventsState.add_five
+        yield YieldEventsState.add_five
+        yield YieldEventsState.add_five 
+
+
+def multiple_yield_example():
+    return rx.cond(
+        YieldEventsState.show_progress,
+        rx.chakra.circular_progress(is_indeterminate=True),
+        rx.heading(
+            YieldEventsState.count,
+            on_click=YieldEventsState.increment,
+            _hover={"cursor": "pointer"},
```

Comment:
Needs a double curly brace to escape - I'm going to update flexdown to not catch these as it comes up too much.